experiments/AtmosGCM/GCMDriver/gcm_bcs.jl

```diff
@@ -21,8 +21,8 @@ function parse_surface_flux_arg(
         boundarycondition = (
             AtmosBC(
                 energy = BulkFormulaEnergy(
-                    (state, aux, t, normPu_int) -> _C_drag,
-                    (state, aux, t) -> bulk_flux(state, aux, t),
+                    (bl, state, aux, t, normPu_int) -> _C_drag,
+                    (bl, state, aux, t) -> bulk_flux(state, aux, t),
```

Comment:
```suggestion
                    (bl, state, aux, t) -> bulk_flux(bl, state, aux, t),
```
I think we'll need this same change across the board

Reply:
Oh wait

Reply:
Maybe not

Reply:
No, it's fine

Reply:
So, I leave it as is?